pom.xml

```diff
@@ -54,7 +54,7 @@
     <httpcomponents.httpclient.version>4.5.11</httpcomponents.httpclient.version>
     <jackson.version>2.10.0</jackson.version>
     <!-- END: AWS jdk version + dependencies -->
-    <hadoop.version>2.7.1</hadoop.version>
+    <hadoop.version>2.8.1</hadoop.version>
```

Comment:
Hive 2.3.7 supposedly depends on Hadoop 2.7.2 so why did you need to move to 2.8.1?

Reply:
Hmm yep I think i did this to fix some Parquet writer error, will revert as this was fixed elsewhere